example/package.json

```diff
@@ -30,7 +29,7 @@
     "react": "16.13.1",
     "react-native": "^0.63.4",
     "react-native-macos": "^0.63.33",
-    "react-native-test-app": "../",
+    "react-native-test-app": "workspace:.",
```

Comment:
iiinteresting - so basically now with Yarn 3 you need to define a local dependency through the workspace?

Reply:
Yeah, what forced the hand was the new checksum feature. It's different every time you ran `yarn install` inside `/example`. Converting to a workspace skips the check on local packages.